### PR TITLE
feature-erms-2718

### PR DESCRIPTION
**Pull request by croqueGrec09**
as of ERMS-2718, the reader number view has been rebuilt according to the specifications, this is a first draft and susceptible to change

### CODE REVIEW

File: app/migrations/changelog-2020-07-24.groovy [hunk 1]
```diff
@@ -0,0 +1,33 @@
+databaseChangeLog = {
+
+	changeSet(author: "galffy (hand-coded)", id: "1595574486444-1") {
```
Comment (Moetez16) [inline app/migrations/changelog-2020-07-24.groovy:3]: hand-coded not modified?

Comment (croqueGrec09) [reply app/migrations/changelog-2020-07-24.groovy:3]: Needs to be changed, I will do so as soon as I can get back to hbz-dev. PR is on hold until that!

Comment (Moetez16) [reply app/migrations/changelog-2020-07-24.groovy:3]: Thanks!

Comment (croqueGrec09) [reply app/migrations/changelog-2020-07-24.groovy]: Changed in 6519ff8

**Pull request merged**